### PR TITLE
DataGrid.Pdf - Simplify column widths declaration

### PR DESCRIPTION
**Pull request by KuznetsovVN**

### CODE REVIEW

File: js/exporter/jspdf/export_data_grid_2.js [hunk 1]
```diff
@@ -26,6 +26,12 @@ function exportDataGrid(doc, dataGrid, options) {
                     if(options.onCellExporting) {
                         options.onCellExporting({ gridCell: { value: cellData.value }, pdfCell });
                     }
+                    if(isDefined(options.columnWidths)) {
+                        const width = options.columnWidths[cellIndex];
+                        if(width && !pdfCell.skip && !pdfCell.rect.w) {
```
Comment (IgnatovDan) [inline js/exporter/jspdf/export_data_grid_2.js]: 'pdfCell.rect.w' can be '0'
let's use '!isDefine(pdfCell.rect.w)'

**Review by IgnatovDan — CHANGES_REQUESTED** [review]

File: js/exporter/jspdf/export_data_grid_2.js (hunk 1 above)
Comment (IgnatovDan) [reply js/exporter/jspdf/export_data_grid_2.js]: And 'if(width' - can be '0'

Comment (IgnatovDan) [reply js/exporter/jspdf/export_data_grid_2.js]: Whether we need a new test?

Comment (KuznetsovVN) [reply js/exporter/jspdf/export_data_grid_2.js]: Made changes to the code and added a test for zero column width

**Review by novsstation — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Review by IgnatovDan — APPROVED** [review]

**Pull request merged**